### DISCUSSION TITLE
Turn off "Show git blame inline" permanently

### DISCUSSION THREAD

**Discussion by volkan0x** [zed-industries/zed#11279] [Feature Requests]
I have to disable the "Show git blame inline" option manually everytime opening a new page. 

Is there any way to disable it permanently or am I doing something wrong?

<img width="1000" alt="Screenshot 2024-05-02 at 01 39 46" src="https://github.com/zed-industries/zed/assets/68243739/67f07ff8-c1a0-4d68-9514-e1a1ed2e6fa9">

**Comment by SomeoneToIgnore** [comment]
There is via the settings, but unfortunately the toggles are not permanent and do not alter the settings (which maybe they should).

Adding this 
```
"git": {
    "inline_blame": {
        "enabled": false
    }
},
```
to your settings.json should remove them until toggled back.
